src/containers/Tables/BidTable/BidTableRow.test.tsx

```diff
@@ -32,7 +30,7 @@ jest.mock(
 );
 
 const renderComponent = ({ data = mockData } = {}) => {
-	return render(
+	return renderWithRedux(
```

Comment:
Does this test need to use `renderWithRedux`? I don't think BidTableRow is using redux, so can probably just use `render`.

Reply:
After introducing row row click navigation to BidTableRow, the existing test fails due to undefined useHistory(). I Used `renderWithRedux` in this case for Router setup.

I could manually wrap this test with `<Router>` but figured cleaner to just use `renderWithRedux`. Thoughts?

Reply:
Makes sense! `renderWithRedux` is generally slower to test with than just `render` - alternatively you could use `render` and mock `useHistory`. I think `renderWithRedux` is all good for now though, as we'll be rewriting these tests in the not-so-distant future.